Yoast/YoastSEO.js
# Implement webworker scheduler

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Added an index.js to the full text tests to export an array of the texts (used with the spam button).
* Added a polling mechanism in the scheduler to check for the next task.

## Test instructions

This PR can be tested by following these steps:

* In the webpack example folder run `yarn` and `yarn start`.
* Use the initialize button first.
* Use the analyze spam button and check the console. The spam requests every test text 10 times. At this point we have 2 texts.
* Check that the worker only runs the last analysis. Or when spamming the spam button, only runs the newest after completion of the last and then always the last one too.

Fixes #1617
